client/packages/purchasing/src/common/Inputs.tsx

```diff
@@ -67,20 +61,21 @@ export const useInputComponents = (
       numericInput: (
         label: LocaleKey,
         value: number | null | undefined,
-        options: NumericInputProps = {}
+        options: NumericTextInputProps = {}
```

Comment:
Allows it to pass through *any* numeric input props, not just a handful of predefined ones.